packages/frontend-test-utils/src/app/createExtensionTester.tsx

```diff
@@ -149,36 +156,52 @@ const AuthenticationProvider = (props: {
   return children;
 };
 
-const TestCoreRouterExtension = createExtension({
+const TestAppRootExtension = createExtension({
   namespace: 'app',
   name: 'root',
   attachTo: { id: 'app', input: 'root' },
   inputs: {
     signInPage: createExtensionInput(
-      {
-        component: createSignInPageExtension.componentDataRef,
-      },
+      { component: createSignInPageExtension.componentDataRef },
       { singleton: true, optional: true },
     ),
     children: createExtensionInput(
-      {
-        element: coreExtensionData.reactElement,
-      },
+      { element: coreExtensionData.reactElement },
       { singleton: true },
     ),
+    elements: createExtensionInput(
+      { element: coreExtensionData.reactElement },
+      { optional: true },
+    ),
+    wrappers: createExtensionInput(
+      { component: createAppRootWrapperExtension.componentDataRef },
+      { optional: true },
+    ),
   },
   output: {
     element: coreExtensionData.reactElement,
   },
   factory({ inputs }) {
     const SignInPage = inputs.signInPage?.output.component;
-    const children = inputs.children.output.element;
```

Comment:
Note for the reviewer: yeah it's unfortunate that this extension is reimplemented over here, but that's just because we don't yet have a mechanism for easily replacing the router. We'll get to that in a later PR